app/services/move_project_to_channel.rb

```diff
@@ -1,13 +1,21 @@
 class MoveProjectToChannel
-  def initialize(project_params,channel_params)
-    @project = Project.find(project_params)
-    @channel = Channel.find(channel_params)
+  def initialize(project_id, channel_id)
```

Comment:
Not really big stuff but you can receive the instances here instead of the ids. So, on the controller you call the Project.find with the id received on the params and, for the channel, I think that there is a method on the ApplicationController which stores the current channel, you can take a look and use this method.